CHANGELOG.md

```diff
@@ -24,6 +25,15 @@ Details
    https://github.com/owncloud/android/issues/3139
    https://github.com/owncloud/android/pull/3184
 
+* Bugfix - Fix a bug when some fields where not retrieved from OIDC Discovery: [#3202](https://github.com/owncloud/android/pull/3202)
+
+   We've fixed a bug when requesting the OIDC discovery. Some fields were handled as mandatory,
```

Comment:
One proposal: i think we should not use the first person to describe the changelog features. Instead of `We've fixed...` i'd set something like: `Problem requesting OIDC discovery was fixed`. What do you think? @abelgardep @JuancaG05 @theScrabi

Reply:
Ok, I can agree with that. But we need to update current tickets too: https://github.com/owncloud/android/blob/master/CHANGELOG.md

Reply:
I agree, better in a passive voice. I was checking some apps in the Play Store and some of them use this new proposed way and others use first person, so I guess both of them are common. Personally I prefer the one @jesmrec said